src/components/interaction-designer/block-editors/MaximumChoicesEditor.vue

```diff
@@ -2,7 +2,7 @@
   <validation-message
     #input-control="{ isValid }"
     :message-key="`block/${block.uuid}/config/maximum_choices`">
-    <div class="block-validation-max-choices">
+    <div class="maximum-choices-editor block-validation-max-choices">
```

Comment:
the same for `block-validation-max-choices`

Reply:
:heavy_check_mark: